src/dashboard/cd-dashboard-news.vue

```diff
@@ -7,8 +7,45 @@
 </template>
 
 <script>
+  import NewsForumsService from './service';
+
   export default {
     name: 'cd-dashboard-news',
+    data() {
+      return {
+        news: null,
+        forums: null,
+      };
+    },
+    computed: {
+      allPosts() {
+        return (this.news).map(post => ({
```

Comment:
Since this computed property gets a bit more complex with further PRs, I think it would be best to split it up into a few props.

I'm struggling to come up with good names. All I can think of are `transformedNews`, `transformedForumPosts` and `allUpdates` or something like that...

Reply:
In addition, instead of using `arr1.concat(arr2)`, nowadays you can just do `[...arr1, ...arr2]`. Either way works fine, and I'm not that picky. Just more of an FYI.

Reply:
Yeah that makes sense. how about formattedNews/Forums ??

Reply:
Yeah, that sounds good